react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
```

Comment:
Your state in the constructor should always have all possible properties defined in it. If state doesn't have starting values it can cause your component to not update correctly.
```JAVASCRIPT
this.state = {
  openId: null
};
```